osbs/build/build_response.py

```diff
@@ -119,6 +119,19 @@ def get_logs(self, decode_logs=True):
         else:
             return logs
 
+    def get_error_message(self):
+        try:
+            str_metadata = graceful_chain_get(self.get_annotations_or_labels(), "plugins-metadata")
+            metadata_dict = json.loads(str_metadata)
+            error = metadata_dict['errors'].items()[0]
```

Comment:
How about `plugin, error = ` since we know items() returns 2-tuples? It might make the next few lines easier to read.